client/src/modules/stock/movements/modals/search.modal.js

```diff
@@ -105,20 +104,19 @@ function SearchMovementsModalController(data, Notify, Instance, Flux, $translate
   vm.cancel = function cancel() { Instance.close(); };
 
   vm.submit = function submit() {
-    var loggedChanges;
 
     // push all searchQuery values into the changes array to be applied
-    angular.forEach(vm.searchQueries, function handleDefinedValue(_value, key) {
-      var _displayValue;
+    angular.forEach(vm.searchQueries, (_value, key) => {
+      let _displayValue;
 
       if (angular.isDefined(_value)) {
         // default to the original value if no display value is defined
         _displayValue = displayValues[key] || lastDisplayValues[key] || _value;
```

Comment:
Unfortunately, this has the bug reported in https://github.com/IMA-WorldHealth/bhima-2.X/issues/2595.  This commit https://github.com/jniles/bhima-2.X/commit/826d7292b5c5832753539f9b9600ffad3d374ec7#diff-a8511c3ae241c3b5dd78e90697b17bc3L113 shows you how to change it so that the bug doesn't exist.